data_pipeline_api/registry/access_upload.py

```diff
@@ -463,65 +508,22 @@ def upload_model_run(
 @click.option(
     "--submission-script", required=True, type=click.Path(exists=True), help="Path to the submission script file.",
 )
```

Comment:
I'm not sure about removing all the remote options and forcing it to all be through env variables, combine them with args taking precedence?

Reply:
+ token - is there any reason to remove these args?

Reply:
no I did actually mean to put them back

Reply:
restored to as they were, which I think is actually what we want?